### PR TITLE
Update AboutUs page with Kai Xuan's info

### PR DESCRIPTION
**Pull request by kaixuan477**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -11,12 +11,11 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
-### John Doe
+### Lim Kai Xuan
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/kaixuan477.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
+[[github](https://github.com/kaixuan477)]
 [[portfolio](team/johndoe.md)]
```
Comment (FooChao) [inline docs/AboutUs.md:19]: Maybe change this as well? Make a team/kaixuan477.md.

File: docs/AboutUs.md [hunk 2]
```diff
@@ -11,12 +11,11 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 
 * Role: Project Advisor
```
Comment (FooChao) [inline docs/AboutUs.md:21]: Role to be updated after our meeting tmr.

File: docs/AboutUs.md (hunk 1 above)
Comment (kaixuan477) [reply docs/AboutUs.md:19]: Noted. Planned to finalise during meeting.

**Review by FooChao — APPROVED** [review]
LGTM!

**Review by awdse22 — APPROVED** [review]
No issues with PR.

**Pull request merged**